lib/bojangles.rb

```diff
@@ -111,6 +137,8 @@ def update_emailed_status!(to:)
   end
 
   def update_log_file!(to:)
-    File.open "#{todays_date}.json", 'w' do |file|
+    File.open (File.join log, "#{todays_date}.json"), 'w' do |file|
```

Comment:
It's nicer to use the parentheses to wrap the method arguments rather than the output of a method, here:

```ruby
File.open File.join(log, "#{todays_date}.json"), 'w' do |file|
```